library/core/src/ptr/const_ptr.rs

```diff
@@ -483,7 +483,7 @@ impl<T: ?Sized> *const T {
     ///
     /// This operation itself is always safe, but using the resulting pointer is not.
     ///
-    /// The resulting pointer "remembers" the [allocated object] that `self` points to; it must not
+    /// The resulting pointer "remembers"[^1] the [allocated object] that `self` points to; it must not
     /// be used to read or write other allocated objects.
```

Comment:
Instead of delegating the term to a footnote, I would suggest adding it as a 2nd sentence.

Reply:
I'm not sure how to do that without disrupting the flow into the second paragraph.  saying "in other words" after an interleaving sentence about terminology seems confusing.

Reply:
Maybe add 'This "memory" that pointers have is referred to as [provenance].' at the end of the paragraph?

Reply:
At the end of which paragraph? I've already said why I don't think it makes sense to be at the end of the first paragraph.

Reply:
Then I would suggest changing the beginning of the 2nd paragraph. A footnote is just too easy to miss to be worth it, IMO.

Reply:
Personally I can't think of some way of phrasing it that would make "<simple explanation>. <tangent>. <verbose explanation>" not confusing.  

Honestly I don't think it's really *worth* more than a footnote, since it's not essential to understanding the invariants of the function, it's more just there for completeness.

Reply:
Not sure what ". ." refers to.

Looking at this in context, I think it'd be fine to just put the sentence here. "In other words" at the beginning of a paragraph can refer to the entire previous paragraph, not just its last sentence.

Reply:
So, to make a concrete proposal:
```
/// The resulting pointer "remembers" the [allocated object] that `self` points to (this is called
/// "[Provenance](ptr/index.html#provenance)"). The pointer must not be used to read or write other allocated objects.
///
/// In other words, ...
```

Reply:
Yeah a parenthetical works. The ".." thing was just some markdown jank.

Reply:
suggestion applied